src/app/frontend/deploy/deploy_controller.js

```diff
@@ -112,6 +117,34 @@ export default class DeployController {
   }
 
   /**
+   * Displays new namespace creation dialog.
+   *
+   * @param {!angular.Scope.Event} event
+   */
+  showNamespaceDialog(event) {
+    this.mdDialog_.show({
```

Comment:
How about moving this object to `namespacfoo_dialog.js` and then `export default` it. This would follow the practice we have now with directives. That template, controller, and definition objects are in separate files.